tests/_utils/view.js

```diff
@@ -0,0 +1,836 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* jshint latedef:false */
```

Comment:
That's to suppress the late class definition warning? I always thought that you can't define a class after it's used (in code). That didn't make sense to me, so I would remove that rule if it's true.